src/style/themes.ts

```diff
@@ -25,7 +25,7 @@ export const darkTheme: DefaultTheme = {
   body: '#141417',
 
   textPrimary: 'rgba(255, 255, 255, 0.8)',
-  textSecondary: 'rgba(255, 255, 255, 0.45)',
+  textSecondary: 'rgba(255, 255, 255, 0.65)',
```

Comment:
This change is causing contrast issues in some places (Badges for instance). 
I'd suggest reverting this for now, and I'll take care of iterating around the styling once the rest of this PR is stabilised. Thanks!